include/mbgl/util/variable_anchor_offset_collection.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
```

Comment:
This should not be in `util`. This directory is for utilities that are useful across the codebase, this code is too specific.